SConstruct

```diff
@@ -8,28 +8,30 @@ from lsst.sconsUtils.utils import libraryLoaderEnvironment
 from lsst.utils import getPackageDir
 from lsst.ci.hsc.validate import *
 
-from SCons import Action
-
-class CallAction(Action.FunctionAction):
-    def __init__(self, ident, func, **kw):
-        self.ident = ident
-        self.func = func
-        Action.FunctionAction.__init__(self, self.func, kw)
-
-    # This function returns what is hashed to generate the function signature.
-    # This signature is what scons uses to decide if the build function has
-    # changed, and thus the target should be rebuilt. Unfortunately functools.partial
-    # or lambda cause scons to generate a different signature on each run, and thus
-    # rebuild everything. This function is overloaded here to ensure that a consistent
-    # function signature is generated
-    def get_contents(self, *args, **kwargs):
-        return str(self.ident)
-
-def makeSCons(cls, root, *args, **kwargs):
-    instance = cls(root)
-    return CallAction(str(args) + str(kwargs), functools.partial(instance.scons, *args, **kwargs))
+from SCons.Script import SConscript
+SConscript(os.path.join(".", "bin.src", "SConscript"))
 
 env = Environment(ENV=os.environ)
+env["ENV"]["OMP_NUM_THREADS"] = "1"  # Disable threading; we're parallelising at a higher level
```

Comment:
Threading is also explicitly disabled in `validate.py` (at least, it would be if that weren't broken). Do you need to do it here as well? It may be that you do, but it would be useful to explain why.